src/Assignment1/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+
+namespace Assignment1
+{
+    /// <summary>
+    /// This is main class <see cref="Program"/>
+    /// </summary>
+    public class Program
+    {/// <summary>
```

Comment:
```suggestion
    {
    
    /// <summary>
```

Add a new line to make code in a good readable format